src/platform/reactnative/scrollcomponent/ScrollComponent.tsx

```diff
@@ -64,12 +65,24 @@ export default class ScrollComponent extends BaseScrollComponent {
                 onScroll={this._onScroll}
                 onLayout={(!this._isSizeChangedCalledOnce || this.props.canChangeSize) ? this._onLayout : this.props.onLayout}>
                 <View style={{ flexDirection: this.props.isHorizontal ? "row" : "column" }}>
-                    <View style={{
-                        height: this.props.contentHeight,
-                        width: this.props.contentWidth,
-                    }}>
-                        {this.props.children}
-                    </View>
+                    {
+                        (!this.props.itemAnimator && this.props.itemContainer) ? (
+                            <ItemContainer style={{
```

Comment:
This big check is not required now

Reply:
Has been refactored to have a contentContainer function